helm/chart-operator-chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the chart-operator
 name: chart-operator-chart
-version: 0.1.6
+version: 0.2.0
```

Comment:
Bumping the minor version so we deploy using a new release channel in cluster-operator.

We need the CRD subresource feature gate to be enabled for 1.10 clusters. So we need to control the roll out.

Reply:
Good point, do we need to update cluster-operator to deploy chart-operator from `0-2-stable`?

Reply:
Yes we'll need to update the chart resource in chart-operator to use the new channel. But first the feature gate needs to be vendored for all providers. 

https://github.com/giantswarm/k8scloudconfig/pull/406